six/common/builtins/datadog_agent.c

```diff
@@ -334,6 +334,7 @@ static PyObject *set_external_tags(PyObject *self, PyObject *args)
         for (j = 0; j < tags_len; j++) {
             PyObject *s = PyList_GetItem(value, j);
             if (s == NULL) {
+                PyErr_Clear();
```

Comment:
is `s == NULL` and the error is set, on paper we have an `IndexError`, does it make sense to `continue`? Wouldn't we just continue having index errors?

Reply:
Adding a `goto error;` sgtm